internal/apis/acme/types_issuer.go

```diff
@@ -641,6 +641,10 @@ type ACMEIssuerDNS01ProviderRFC2136 struct {
 	// Supported values are (case-insensitive): ``HMACMD5`` (default),
 	// ``HMACSHA1``, ``HMACSHA256`` or ``HMACSHA512``.
 	TSIGAlgorithm string
+
+	// Protocol to use for DNS queries. Valid values are (case-sensitive) ``tcp`` and ``udp``; ``udp`` (default).
+	// +optional
+	Protocol string `json:"protocol,omitempty"`
```

Comment:
```suggestion
	Protocol string
```

No need for these things in the internal API structs.